translations/Polish/special-terms.md

```diff
@@ -0,0 +1,41 @@
+# Terms
+English word | Polish word 
+
+* Message - Wiadomość
+* Private Message (PM) - Prywatna wiadomość
+* Group PM - Czat grupowy (Konwersacja would be too formal)
+* Realm - Domena (Królestwo wouldn't fit the context)
+* Stream - Kanał (Strumień wouldn't fit the context)
+* Topic - Wątek 
+* Private/Invite-Only Stream - Prywatny kanał (Kanał na zaproszenie seems too long)
+* Public Stream - Publiczny kanał
+* Bot - Bot (Robot just doesn't sound right in my opinion)
+* Integration  - Integracja
+* Notification - Powiadomienie
+* Alert Word - Słowo alarm
```

Comment:
I think this one is too literal - alert words are words you get notifications for, just like you'd for an @-mention. Something more like "oznaczone/otagowane słowo" would probably convey the meaning better.